common/static/common/js/spec/discussion/view/discussion_thread_list_view_spec.js

```diff
@@ -542,5 +543,21 @@
             });
         });
 
+        describe('thread preview body', function() {
+            it('thread should have preview body when noThreadBodyPreview is false', function() {
```

Comment:
Nit: test names should be read starting with ``it``, so this reads as ``it thread should have preview body...``. I'd change this to something like ``it('should be shown when...``, because the it refers to ``thread preview body``.